expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
I think there's a couple things flipped here. For an adaptive track, down is harder, so it goes down on correct answers and up on incorrect. So at the minimum, if you get it wrong, it goes up (regardless of ``repeat_limit``.). If you get it right, it will count a reversal and change the direction to up.